src/main/java/de/tum/in/www1/artemis/web/rest/ExerciseResource.java

```diff
@@ -242,12 +223,15 @@ else if (authCheckService.isAtLeastTeachingAssistantForExercise(exercise, user))
     }
 
     /**
-     * Reset the exercise by deleting all its participations /exercises/:exerciseId/reset This can be used by all exercise types, however they can also provide custom implementations
+     * DELETE exercises/:exerciseId/participations
+     *
+     * Reset the exercise by deleting all its participations
+     * This can be used by all exercise types, however they can also provide custom implementations
      *
      * @param exerciseId exercise to delete
      * @return the ResponseEntity with status 200 (OK)
      */
-    @DeleteMapping(value = "/exercises/{exerciseId}/reset")
+    @DeleteMapping("exercises/{exerciseId}/participations") // TODO: should be courses/{courseId}/exercises/{exerciseId}/participations
```

Comment:
I'd prefer if we keep /reset.
I'd rather change to PostMapping then.
I think reset does a bit more than "just" deleting participations even if the java doc might state it differently